presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator resource group aggregation issue related to WAITING_FOR_PREREQUISITE queries.
+* Fix Iceberg $files table in case of column dropping.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory overcounting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an error in ``tdigest_agg``.
+* Add `to_base32` and `from_base32` varbinary functions.
+* Add support for TRUNCATE TABLE statement.
+* Add support to set timeout on the query analyzer runtime to prevent long running query analysis like complex regular expressions. The timeout duration can be set by the configuration property `planner.query-analyzer-timeout` or session property `query_analyzer_timeout`.
+* Add trim_array function.
+* Added a new optimization for showing results for (interactive) distinct limit N as they become available with no buffering.  This can be enabled using the session param: quick_distinct_limit_enabled to true.
+* Removed state (PageBuilder) in ZIP_WITH as it was getting clobbered when used in REDUCE_AGG context.
+* ...
+* ...
+* ...
+* Bump postgresql driver to 42.3.3.
+* Change  protoc and grpc version in presto-grpc-api module to adapt linux aarch64.
+* If hive files are changed, the modification time can be used to invalidate parquet metadata cache immediately.
+* Upgrade zstandard compression to version 1.5.2.2. This can improved about 2% overall CPU with data compressed with zstandard.
+* We have an option to now to do a local round robin shuffle to reduce the partial distinct limit output size.
+
+Verifier Changes
+________________
+* Add support to verifier for running control and test queries concurrently through config `concurrent-control-and-test` and test property.
+
+SPI Changes
+___________
+* Add `getInfoMap` method in `ConnectorSplit` which returns a `Map<String, String>`. This method should be preferred to the `getInfo` method which returns a raw object.
+
+Hive Changes
+____________
+* Fix Cassandra documentation for supported partition types.
+* Add a limit that prevents caching partitions with column counts greater than a configured threshold. This threshold can be set using the `hive.partition-cache-column-count-limit` configuration parameter.
+* Add metastore config property ``hive.metastore.thrift.delete-files-on-table-drop`` to delete files on drop table (:pr:`17369`).
+* ...
+
+Iceberg Changes
+_______________
+* Add $properties system table.
+* Add schema evolution in the iceberg doc.
+* Add time travel in the iceberg doc.
+* Store column comments for iceberg tables.
+* Upgrade Iceberg to 0.13.1.
+
+Mongodb Changes
+_______________
+* Add CAST(ObjectId() as STRING).
+
+Pinot Changes
+_____________
+* Always double-quotes identifiers in generated Pinot queries.
```

Comment:
This sentence seems to be broken. Also I'm not sure what is the user impact here.